src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -254,6 +254,8 @@ def __init__(
 
     def build(self) -> SlackBody:
         # XXX(dcramer): options are limited to 100 choices, even when nested
+        if self.event:
+            self.event = self.event.for_group(self.event.group or self.event.groups[0])
```

Comment:
@wedamija is this safe to do or will unexpected things break in prod? I can't actually replicate the problem locally so it's a bit tough to debug.

Reply:
We shouldn't do this here, at the point we're in the message builder we should expect that we already have a `GroupEvent`. If there are callers that are passing `Event` then we should change this at that level instead.